scripts/mockery_generate.sh

```diff
@@ -2,5 +2,8 @@
 #
 # Invoke Mockery v2 to update generated mocks for the given type.
 
-go run github.com/vektra/mockery/v2 --disable-version-string --case underscore --name "$*"
+
+
+go install github.com/vektra/mockery/v2@latest
+mockery --disable-version-string --case underscore --name "$*"
```

Comment:
```suggestion
go run github.com/vektra/mockery/v2@latest --disable-version-string --case underscore --name "$*"
```
Consider this alternative